Library/Homebrew/livecheck/strategy/pypi.rb

```diff
@@ -1,19 +1,21 @@
 # typed: strict
 # frozen_string_literal: true
 
+require "json"
+require "utils/curl"
+
```

Comment:
```suggestion
```

* We use `Json` strategy methods within strategies that parse JSON, as in the [`Crate` strategy](https://github.com/Homebrew/brew/blob/405cedad235fa9d6960e19b78705d0f43094b140/Library/Homebrew/livecheck/strategy/crate.rb).
* We use `Strategy#page_content` (or `Strategy#page_headers`) to fetch content within strategies.